.circleci/config.yml

```diff
@@ -235,7 +235,7 @@ jobs:
     docker:
       # specify the version you desire here
       # - image: circleci/python:3.8.0
-      - image: gethue/hue:latest-py2
```

Comment:
Why are we changing docker image of `build` CI check? This job checks for Hue py2 build.